data/formats-data.js

```diff
@@ -5087,7 +5094,7 @@ exports.BattleFormatsData = {
 		tier: "PU",
 	},
 	mienfoo: {
-		randomBattleMoves: ["uturn", "drainpunch", "stoneedge", "swordsdance", "batonpass", "highjumpkick", "fakeout", "knockoff"],
+		randomBattleMoves: ["uturn", "drainpunch", "poisonjab", "swordsdance", "batonpass", "highjumpkick", "fakeout", "knockoff", "bounce", "acrobatics"],
```

Comment:
stone edge > bounce (larvesta, ponyta)

Reply:
>Z-bounce?

Reply:
with the presence of knock off, acrobatics is a reusable option that does good damage with similar coverage to zbounce

Reply:
would it be better to just kill bounce?